include/inviwo/core/io/serialization/deserializer.h

```diff
@@ -364,6 +365,12 @@ class ContainerWrapper {
 
 namespace util {
 
+template <typename T>
```

Comment:
We might want to move these functions to another namespace, detail?

Reply:
yes

Reply:
will move in the next pr...